lib/cloud_controller/blobstore/cli/azure_cli_client.rb

```diff
@@ -0,0 +1,145 @@
+require 'open3'
+require 'tempfile'
+require 'fileutils'
+require 'cloud_controller/blobstore/base_client'
+require 'cloud_controller/blobstore/cli/azure_blob'
+
+module CloudController
+  module Blobstore
+    # POC: This client uses the `azure-storage-cli` tool from bosh to interact with Azure Blob Storage.
+    # It is a proof of concept and not intended for production use.
+    # Goal of this POC is to find out if the bosh blobstore CLIs can be used as a replacement for the fog.
```

Comment:
Any idea if the bosh blobstore CLIs have consistent interfaces?  Do you think that in the future we could have something like one client that takes the provider CLI as an injection?